### DISCUSSION TITLE
分享一个 SIGSOFT 的实证研究方法标准

### DISCUSSION THREAD

**Discussion by xiaoya-yaya** [X-lab2017/open-wonderland#214] [3️⃣ 聊学术]
https://acmsigsoft.github.io/EmpiricalStandards/docs/

这份材料包含了软件工程领域各类**研究方法**的研究标准和经典参考文献，包括 benchmarking、case study、grounded theory、repository mining...是我们选取研究方法时的一个好参考。

**Comment by bifenglin** [comment]
强的，周四讲一下？

**Reply by xiaoya-yaya** [reply]
可以先安排着，来得及的话就讲～
